packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-mixin.js

```diff
@@ -428,8 +423,27 @@ export const MultiSelectComboBoxMixin = (superClass) =>
         },
       });
       this.addController(this._overflowController);
+    }
 
-      this.__updateChips();
```

Comment:
Calling `__updateChips()` in `ready()` is actuallly unnecessary - chips will be updated by `updated()` which gets called immediately after `ready()`, also they will be always updated on initial resize using `_onResize()`.